src/transformers/models/hiera/__init__.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import TYPE_CHECKING
+
+from ...utils import (
+    OptionalDependencyNotAvailable,
+    _LazyModule,
+    is_torch_available,
+)
+
+
+_import_structure = {
+    "configuration_hiera": [
+        "HIERA_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "HieraConfig",
+    ],
+}
+
+try:
+    if not is_torch_available():
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    pass
+else:
+    _import_structure["modeling_hiera"] = [
+        "HIERA_PRETRAINED_MODEL_ARCHIVE_LIST",
+        "HieraModel",
+        "HieraPreTrainedModel",
+        "HieraBlock",
```

Comment:
@Namangarg110 The error is happening because `"HieraBlock"` is being aded here but isn't included in `src/transformers/__init__.py`. It should be removed here.

For the other tests, make sure to click through to circleci to see the errors and what needs to be addressed.

Reply:
Thankyou for swift response @amyeroberts. I already tried removing ```"HieraBlock"``` from ```src/transformers/models/hiera/__init__.py```. However, I am still getting the following difference: 

```
torch backend:
  HIERA_PRETRAINED_MODEL_ARCHIVE_LIST in TYPE_HINT but not in _import_structure.
  HieraModel in TYPE_HINT but not in _import_structure.
  HieraPreTrainedModel in TYPE_HINT but not in _import_structure.
Differences for tf backend:
  HIERA_PRETRAINED_MODEL_ARCHIVE_LIST in _import_structure but not in TYPE_HINT.
  HieraModel in _import_structure but not in TYPE_HINT.
  HieraPreTrainedModel in _import_structure but not in TYPE_HINT.
make: *** [Makefile:42: repo-consistency] Error 1
```

Seems like ```HIERA_PRETRAINED_MODEL_ARCHIVE_LIST, HieraModel, HieraPreTrainedModel``` are somehow getting into tf's ```__import_structure```

Reply:
Thank you, @amyeroberts. I apologize for the oversight.

I've made the necessary corrections, and now all tests are passing except for one:

`transformers/tests/test_modeling_utils.py:test_use_safetensors` is failing. I believe this should not be related to my changes, but could it be?

Reply:
@Namangarg110 Indeed - that isn't related to this PR. You can ignore it for now. I'll find out if it's something we've addressed upstream